docs/development/docs.rst

```diff
@@ -5,32 +5,77 @@ As one might expect,
 the documentation for Read the Docs is built using Sphinx and hosted on Read the Docs.
 The docs are kept in the ``docs/`` directory at the top of the source tree.
 
-.. TODO: expand this section explaining there the PR is automatically built and
-   the author can visualize changes without installing anything on their system.
-   However, if there is going to be periodic/bigger contributions, it may be a
-   good idea to install the Sphinx requirements to build our docs.
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation,
+you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+#. click the checkmark next to your commit and it will expand to have multiple options
+
+#. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+
+   .. image:: /img/details_link.png
+
+#. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+.. note::
+
+    As of April 2021, you must use python 3.6 to generate the documentation.
```

Comment:
I think we can remove the date from this warning, we can update this note when we update our code base and tests to support another version of python (3.9)